pkg/services/store/object/object.proto

```diff
@@ -3,51 +3,65 @@ package object;
 
 option go_package = "./;object";
 
-// The canonical object/document data -- this represents the raw bytes and storage level metadata
-message RawObject {
-  // Unique ID
-  string UID = 1;
+message GRN {
```

Comment:
I think the original long form of the name was _grafana resource name_, hence GRN. Do we want to stick with "GRN"?:)

Reply:
After playing with other options, this is the best for now -- we will still encode this in the string form when that makes sense though